array-methods.js

```diff
@@ -5,23 +5,35 @@ var dataset = require('./dataset.json');
   greater than 100000
   assign the resulting new array to `hundredThousandairs`
 */
-var hundredThousandairs = null;
+var hundredThousandairs = dataset.bankBalances.filter(function (element) {
+  return element.amount > 100000;
+});
 
 // set sumOfBankBalances to be the sum of all value held at `amount` for each bank object
-var sumOfBankBalances = null;
+var sumOfBankBalances = dataset.bankBalances.map(function (element) {
+  return element.amount;
+}).reduce(function (previous, current) {
```

Comment:
The normal pattern for this is to put the `.reduce` or any chained function on the next line and indented in two spaces. Please update the rest of the file to follow that convention.

Reply:
Line 13 can stay the way that it is, it's usually any chained function after the first one.